.github/workflows/versioning.yml

```diff
@@ -0,0 +1,31 @@
+name: Versioning RC
+
+on:
+  workflow_run:
+    workflows: ["Running unittest"]
+    branches: [master]
+    types: 
+      - completed
+
+jobs:
+  versioning-auto-commit:
```

Comment:
so this will bump the version upon any merge to master.
and it either take a tag or bumps a minor/patch?

Reply:
Yes, any merge to master will produce: **0.0.1rc5-dev1**, **0.0.1rc5-dev2**, **0.0.1rc5-dev3**
It doesn't take a tag. The tag is the release like **0.0.1rc5** and we will add **-dev1**